### PR TITLE
[Communication][Email] Separating Email test resources + Adding PPE live tests

### PR DESCRIPTION
**Pull request by yogeshmo**
### Packages impacted by this PR
@azure/communication-*

### Describe the problem that is addressed by this PR
- Adding PPE SDK live tests for the email communication package.
- Follow up to https://github.com/Azure/azure-sdk-for-js/pull/24175. That PR added email resource creation to a common resources template shared by all communication tests. However, only `communication-email` package needs the email resource for its test pipeline. So this PR adds an email-specific resource template, while reverting changes to the common template.

### CODE REVIEW

File: sdk/communication/CONTRIBUTING.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ If you are testing against a personal resource, you can check the [Managed Ident
 
 For a more in-depth look on how to authenticate using managed identity, refer to the [Azure Identity client library for Java](https://docs.microsoft.com/java/api/overview/azure/identity-readme) documentation. This document also has more ways for you to authenticate using the DefaultAzureCredential object besides the ones we discussed in this contributing file.
 
+### Running the New-TestResources and Remove-TestResources Scripts
+
+You may want to run the `New-TestResources.ps1` or `Remove-TestResources.ps1` scripts to test resource deployments in the SDK Live Test pipelines locally. You will need to add the additional `-TestResourceDirectories` parameter when running these scripts. If you are using the shared `test-resources.json` you can point to `communication/test-resources/`. If you are using a custom `test-resources.json`, you can point to `communication/<package-name>/test-resources.json`.
```
Comment (benbp) [inline sdk/communication/CONTRIBUTING.md:58]:
```suggestion
You may want to run the `New-TestResources.ps1` or `Remove-TestResources.ps1` scripts to test resource deployments in the SDK Live Test pipelines locally. You will need to add the additional `-TestResourcesDirectory` parameter when running these scripts. If you are using the shared `test-resources.json` you can point to `communication/test-resources/`. If you are using a custom `test-resources.json`, you can point to `communication/<package-name>/test-resources.json`.
```

The yaml does the loop, the script parameter is singular.

**Review by benbp — APPROVED** [review]

**Review by kagbakpem — APPROVED** [review]

**Pull request merged**